compiler/luci/pass/src/FuseAddWithFullyConnectedPass.cpp

```diff
@@ -24,6 +26,49 @@
 
 namespace
 {
+
+#define RETURN_FALSE_UNLESS(cond) \
+  if (not(cond))                  \
+    return false;
+
+inline bool common_pass_checks(const loco::DataType dtype, luci::CircleAdd **add,
```

Comment:
Why did you use `inline`?

Reply:
Could you rename this to `fc_with_add_pattern_check`?

Reply:
@jinevening yes sure I can change it.

PS I am really sorry for a really late response, I was on a longer sick leave.